align_anything/configs/evaluation/vllm/vllm_basic.json

```diff
@@ -13,7 +13,8 @@
     {
         "tokenizer_mode": "auto",
         "trust_remote_code": false,
-        "gpu_memory_utilization": 0.6,
+        "gpu_memory_utilization": 0.3,
+        "tensor_parallel_size": 1,
```

Comment:
Setting "tensor parallel size" to 1 is too slow, you can leave it unset and use the default value.